Project/Source/Components/UI/ComponentTransform2D.cpp

```diff
@@ -121,21 +159,29 @@ void ComponentTransform2D::Load(JsonValue jComponent) {
 	JsonValue jPivot = jComponent[JSON_TAG_PIVOT];
 	pivot.Set(jPivot[0], jPivot[1]);
 
+	JsonValue jPivotPosition = jComponent[JSON_TAG_PIVOT_POSITION];
+	pivotPosition.Set(jPivotPosition[0], jPivotPosition[1], jPivotPosition[2]);
+
 	JsonValue jSize = jComponent[JSON_TAG_SIZE];
 	size.Set(jSize[0], jSize[1]);
 
 	JsonValue jAnchorX = jComponent[JSON_TAG_ANCHOR_X];
-	anchorX.Set(jAnchorX[0], jAnchorX[1]);
+	anchorMin.Set(jAnchorX[0], jAnchorX[1]);
 
 	JsonValue jAnchorY = jComponent[JSON_TAG_ANCHOR_Y];
-	anchorY.Set(jAnchorY[0], jAnchorY[1]);
+	anchorMax.Set(jAnchorY[0], jAnchorY[1]);
 
 	dirty = true;
 }
 
 void ComponentTransform2D::DrawGizmos() {
+	ComponentCanvasRenderer* canvasRenderer = GetOwner().GetComponent<ComponentCanvasRenderer>();
+	float factor = canvasRenderer->GetCanvasScreenFactor();
 	if (!App->time->IsGameRunning()) {
 		dd::box(GetPosition(), dd::colors::Yellow, size.x * scale.x / 100, size.y * scale.y / 100, 0);
+		float3 pivotPosFactor = float3(GetPivotPosition().x * factor / 100, GetPivotPosition().y * factor / 100, GetPivotPosition().z * factor / 100);
+		dd::box(pivotPosFactor, dd::colors::OrangeRed, 0.1, 0.1, 0);
+
```

Comment:
Add functionality do debug this ingame. References in DrawGizmos of other components

Reply:
Ok! I will search how to do it with the references, but I don't understand why is useful this functionality for debuging ingame